models/fates/R/met2model.FATES.R

```diff
@@ -61,103 +67,119 @@ met2model.FATES <- function(in.path, in.prefix, outfolder, start_date, end_date,
       
       ## extract variables. These need to be read in and converted to CLM names (all units are correct)
       time      <- ncvar_get(nc, "time")
-      latitude  <- ncvar_get(nc, "latitude")
-      longitude <- ncvar_get(nc, "longitude")
+      LATIXY    <- ncvar_get(nc, "latitude")
```

Comment:
```suggestion
      LATIXY    <- ncdf4::ncvar_get(nc, "latitude")
```

Reply:
Also, I'm not sure what's gained by changing variable names here. We generally find that the code is easier to understand when the internal variable names match the standard names they are extracting.

Reply:
Hi @mdietze , the changed variable names like LATIXY, LONGXY here were intended to be consistent with those names in fates model.

Reply:
General preference for the PEcAn code is to use standard names internally in the code, and then match those to model-specific names when writing them out, unless there's some specific computational advantage in doing so (e.g. code that parses the internal names to write to output)